subscription/upgrade.md

```diff
@@ -7,29 +7,29 @@ redirect_from:
 - /docker-hub/billing/upgrade/
 ---
 
-This page contains information on how to purchase a Docker paid subscription. It also contains information on how to upgrade your existing subscription and update your billing information.
+When you upgrade to a paid subscription, you immediately have access to all the features and entitlements available in your new chosen subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
 
-You can upgrade from Docker Personal to a Pro, Team, or a Business subscription from a Personal subscription. When you upgrade to a paid subscription, you can immediately use all the features and entitlements offered in your new subscription. For detailed information on features available in each subscription, see [Docker Pricing](https://www.docker.com/pricing){: target="_blank" rel="noopener" class="_"}.
+## Upgrade from a free subscription to a paid subscription 
 
-## Upgrade to a Pro, Team, or Business subscription
+1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
 
-The following section contains information on how to purchase a Docker  subscription for new customers. To upgrade from a Personal subscription to a Team or Business subscription, you must first convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
+2. Convert your personal namespace to an organization namespace. To learn more, see [convert an account into an organization](../docker-hub/convert-account.md).
 
-To purchase a Pro, Team, or Business subscription:
+3. From the **Billing** section, select the **Plan** tab.
 
-1. Log into your [Docker Hub](https://hub.docker.com){: target="_blank" rel="noopener" class="_"} account.
+4. Select **Change plan** and then choose the plan you'd like to upgrade to. 
 
-2. To upgrade your Personal subscription, click **Upgrade**. To upgrade your Pro or Team subscription, open the drop-down menu next to your username in the top-right corner and select **Billing**. On the **Billing Details** page, click **Change plan** to navigate to the pricing page.
+5. Follow the on-screen instructions.
 
-   > **Note**
-   >
-   > If you haven't created an organization for a Team or Business subscription, select **Create Organization** from the drop-down menu and select your plan. If you already have a subscription, you can also add another organization. Select **Buy Now** and continue with the on-screen instructions. The Team subscription is now a 100 seat limit. For more seats, you can upgrade to a Business subscription.
+## Upgrade your paid subscription
```

Comment:
**nitpick:** I don't think there are different steps between upgrading from a free subscription to a paid subscription or updating the existing paid subscription. However, there is a difference between upgrading a brand-new account and an existing account. 

If a customer already has a Docker user or organization account, s/he should navigate to the Billing section (https://hub.docker.com/billing).

If a customer doesn't have a Docker user account, s/he should sign up for a new one (https://hub.docker.com or https://hub.docker.com/signup) and choose between Persona and Pro (https://hub.docker.com/choose-plan). 

If a customer has a Docker user account but doesn't have a Docker organization account, s/he should create a new organization (https://hub.docker.com/orgs) and choose between Team and Business (https://hub.docker.com/choose-plan?type=organization).